massdriver-application/main.tf

```diff
@@ -18,7 +19,10 @@ locals {
 
   policies = { for p in local.app_policies : p => jsondecode(data.jq_query.policies[p].result) }
 
-  base_envs = { for k, v in local.app_envs : k => jsondecode(data.jq_query.envs[k].result) }
+  base_envs = merge(
```

Comment:
merge in HCL handles collisions by prioritizing the last specified value, so here we are prioritizing params from the UI over those in the app definition.